config/dockerfiles/Dockerfile.Rust

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
```

Comment:
I agree.  There is not.  There doesn't seem to be a plan for one.  This is Dockerfile is a good example for other future languages we may want to add in without official Docker repos.

There are some Rust Docker images available but they're not official.  I think we need a high level of trust for the images we use.